pyproject.toml

```diff
@@ -10,22 +10,13 @@ skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 [tool.coverage.run]
 branch = true
 omit = [
-    # These are covered by `tests/integration/test_runner_manager_openstack.py`.
-    "src/openstack_cloud/openstack_cloud.py",
-    "src/openstack_cloud/openstack_runner_manager.py",
-    # Contains interface for calling LXD. Tested in integration tests and end to end tests.
-    "src/lxd.py",
     # Contains interface for calling repo policy compliance service. Tested in integration test
     # and end to end tests.
     "src/repo_policy_compliance_client.py",
-    # TODO: 2024-04-17: These files are pending a major refactor. The refactor includes a RunnerManager
-    # interface class which will include a complete re-organization of the code in these files.
-    "src/runner.py",
-    "src/runner_manager.py",
 ]
 
 [tool.coverage.report]
-fail_under = 84
+fail_under = 83
```

Comment:
Normally we don't allow coverage drops but I assume this is OK for this kind of PR.

Reply:
Not sure where it will finish :). After removing the InstanceType class it is back in 84%.... I will adjust it after the PR is approved...